chainer/training/triggers/early_stopping_trigger.py

```diff
@@ -0,0 +1,126 @@
+from chainer import reporter
+from chainer.training import util
+
+
+import chainer
+import operator
+import warnings
+
+
+class EarlyStoppingTrigger(object):
+    """Trigger for Early Stopping.
+    This trigger works only when it could monitor
```

Comment:
Add one blank line after the summary line. I also think the explanation is still not enough and ambiguous, so I suggest the following revised version.

```
This trigger fires when the model is not improved for specified periods.
It can be used as a stop trigger of :class:`~chainer.training.Trainer`  to realize *early stopping* technique.

This trigger works as follows.
Within each *check interval* defined by the ``check_trigger`` argument, it monitors and accumulates the reported value at each iteration.
At the end of each interval, it computes the mean of the accumulated values and compares it to the previous ones to maintain the *best* value.
When it finds that the best value is not updated for some periods (defined by `patients`), this trigger fires.
```
Please insert newlines appropriately to keep flake8 calm.

Reply:
Thanks for suggestion, I use it.. :bow: